docs/developer/release_process.rst

```diff
@@ -0,0 +1,124 @@
+.. # Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. # other BLT Project Developers. See the top-level LICENSE file for details
+.. # 
+.. # SPDX-License-Identifier: (BSD-3-Clause)
+
+Release Process
+===============
+
+.. note:: No significant code development is performed on a release branch.
+          In addition to preparing release notes and other documentation, the
+          only code changes that should be done are bug fixes identified
+          during release preparations
+
+Here are the steps to follow for an BLT release.
+
+1: Start Release Candidate Branch
+---------------------------------
+
+Create a release candidate branch off of the develop branch to initiate a
+release. The name of a release branch must contain the associated release version
+number. Typically, we use a name like v0.4.0-rc
+(i.e., version 0.4.0 release candidate).
+
+.. code:: bash
+
+    git checkout -b v0.4.0-rc
+
+2: Update Versions in Code
+--------------------------
+
+**Update BLT_VERSION**
+
+   * ``SetupBLT.cmake``: ``set(BLT_VERSION "0.4.0" CACHE STRING "")``
+
+
+**Update Release Notes**
+
+#. Update ``RELEASE-NOTES.md`` by changing the 
+   ``unreleased`` section from:
+
+.. code:: none
```

Comment:
This is missing from the docs.

I guess it doesn't like `none` for the language?